bin/improver-generate-topographybands-ancillary

```diff
@@ -41,7 +41,12 @@ from improver.generate_ancillaries.generate_ancillary import (
 from improver.generate_ancillaries.generate_ancillary import (
      find_standard_ancil)
 
-
+# The following dictionary defines the orography bands masks are required for.
+#   'land': land points outside the band are set to 0 and those within
```

Comment:
Bands of gold? Laurel? I'm not sure 'orography bands' indicates that the thresholds are altitudes. You do say just below 'above the thresholds' but it would be great to indicate somewhere that we are talking about the vertical, that these are bands of altitude above/below mean sea level.